grails-app/jobs/com/causecode/fileuploader/DailyJob.groovy

```diff
@@ -20,17 +22,23 @@ class DailyJob {
     }
 
     def execute() {
-        log.info 'Started executing DailyJob..'
-        fileUploaderService.renewTemporaryURL()
-        fileUploaderService.moveFailedFilesToCDN()
+
+        boolean renewJobDisabled = Holders.config.jobs.fileUploader.renewURLs.disable ?: false
```

Comment:
disable -> disabled
renewJobDisabled -> isDisabled

Reply:
isRenewUrlJobDisabled was better if we using a method here.